Sharpmake/PackageReferences.cs

```diff
@@ -158,14 +158,16 @@ public enum AssetsDependency : uint
             None = 0,
             Compile = 1 << 0,
             Runtime = 1 << 1,
+            [Obsolete("Use 'ContentFiles' insead")]
```

Comment:
typo: insead

Reply:
sorry, I need some coffee xD

Reply:
What do you think about adding **buildMultitargeting** and **buildTransitive** as well with this PR?

Reply:
Sure! or as a new PR, that way we don't mix the commits :)

Reply:
Alright, will make a follow up PR for the other two enum values. For the other types of AssetDependencies I dont know how to approach implementing support for those.